WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -48,6 +50,9 @@
             </td>
           <% end %>
 
+          <td>
+            <%= registration.events.length %>
```

Comment:
Hi @jonatanklosko, does it make a difference if the view has already fetched the whole events from the database and mapped them with the Event object? The view already does `<% if registration.events.include?(event) %>`

Or is it still better to use the `registration.registration_events.count`?